modules/configmap/main.tf

```diff
@@ -28,55 +28,3 @@ data "google_project" "project" { project_id = var.project_id }
 // What identity is deploying this?
 data "google_client_openid_userinfo" "me" {}
 
-// Create an alert policy to notify if the secret is accessed by an unauthorized entity.
```

Comment:
secret related, maybe keep?

Reply:
this is modules/configmap, so secret isn't really secret

Reply:
we have so many of these in dev envs and it will multiply

Reply:
you sure its not a secret? line 17 of this file is a secret in the secret manager

for dev, i think we want a knob to enable to turn on these alert policies, off by default, turn on for staging and prod

Reply:
IIRC using secrets here is an implementation detail, they are not considered to actually be secret. Those would use the secret module

Reply:
Yes, it is a "secret" because there is no ConfigMap object in CloudRun, and we have to mount using Secret. But both the module and the alert says "abnormal ConfigMap access"

Reply:
Yeah, ok to kill `configmap` secret stuff.